docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+    web:
+        build: .
+        volumes:
+            - .:/code/src
+        command: >
+            bash -c "gulp & ./yii serve 0.0.0.0"
```

Comment:
Are you sure serve can handle all the resources we have?

Reply:
Sorry, what do you mean?

Reply:
`./yii serve` uses PHP's internal server that is:

1. Handles URLs a bit differently.
2. Is single-threaded and locking when it comes to serving resources.

Both are likely OK for development.

Reply:
Got it now. Haven't noticed any problems yet.